lib/js/components/Buttons/IconButton/IconButton.vue

```diff
@@ -18,7 +23,14 @@
 		<div
 			v-if="$slots.default && type !== ICON_BUTTON_TYPES.ICON_ONLY"
 			class="a-iconButton__label"
-			:class="{ '-minor': colorScheme === ICON_BUTTON_COLOR_SCHEMES.MINOR_LABEL }"
+			:class="{
+				'-neutral':
+					colorScheme === ICON_BUTTON_COLOR_SCHEMES.NEUTRAL_LABEL &&
+					color !== ICON_BUTTON_COLORS.NEUTRAL_WEAK,
```

Comment:
why do we exclude `netural-weak` here?

Reply:
does UX plan to use `NEUTRAL_LABEL` with `NEUTRAL_WEAK`? In figma I see `ALL_IN_COLOR` in such cases. If we agree that UX uses `ALL_IN_COLOR`, then we get no exception in the code => better maintenability

Reply:
According to Figma: https://www.figma.com/file/izQdYyiBR1GQgFkaOIfIJI/LMS---DS---Components?node-id=2770%3A62766
All cases with gray label has always three tokens: `$color-neutral-text`, `$color-neutral-text-hovered` , `$color-neutral-text-focused`, `$color-neutral-text-disabled`, `neutral-weak` is outdated. So condition looks simple now:
```
:class="{
	'-neutral': colorScheme === ICON_BUTTON_COLOR_SCHEMES.NEUTRAL_LABEL,
}"
```